Library/Formula/meanwhile.rb

```diff
@@ -0,0 +1,299 @@
+class Meanwhile < Formula
+  desc "Implementation of Lotus Sametime Community Client protocol"
+  homepage "http://meanwhile.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/meanwhile/meanwhile/1.0.2/meanwhile-1.0.2.tar.gz"
+  sha256 "38a8059eaa549cbcb45ca0a5b453e9608ceadab360db2ae14581fb22ddabdf23"
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  patch :DATA
+
+  def install
+    ENV.deparallelize
```

Comment:
I'm not quite sure what you are looking for here. Nevertheless, as I mentioned above, the project doesn't seem to be maintained. Please let me know if there's anything I should add to this proposed pull request.